### PR TITLE
♿️ - Settings - Notifications Heading

### PR DESCRIPTION
**Pull request by cdolm92**
# 📲 What

`SettingsHeaderView` text in notifications now read as a header on VoiceOver.
# 🤔 Why

Users can navigate through this screen by section.

# 🛠 How

adding  ` _ = self
      |> \.accessibilityLabel .~ title
      |> \.accessibilityTraits .~ .header
      |> \.isAccessibilityElement .~ true`

# ✅ Acceptance criteria
on all the following: iOS 10, iOS 11 & iOS 12
with Voice Over turned ON

- [ ] Navigate to `Settings > Notifications`
- [ ] Tap first heading it should read `Projects you've backed - heading`
- [ ] Test w/ [Rotor Menu](https://support.apple.com/en-us/HT204783): Set it to `Headings`, swipe up or down making sure all heading titles are read followed by `"heading"`

# Todo
- [x] Clean up PR

### CODE REVIEW

File: Kickstarter-iOS/Views/Cells/SettingsAccountWarningCell.swift [hunk 1]
```diff
@@ -14,13 +14,6 @@ final class SettingsAccountWarningCell: UITableViewCell, ValueCell, NibLoading {
 
     _ = self.warningIconImage
       |> \.isHidden .~ shouldHideAlertIcon
-
-    if !shouldHideAlertIcon {
```
Comment (dusi) [inline Kickstarter-iOS/Views/Cells/SettingsAccountWarningCell.swift]: Was this done by an accident? It effectively reverts changes done in https://github.com/kickstarter/ios-oss/pull/535/files 😄

Comment (cdolm92) [reply Kickstarter-iOS/Views/Cells/SettingsAccountWarningCell.swift]: Had a merge conflict, going to correct that now

**Comment by dusi** [comment]
👍 works on my machine but would appreciate @justinswart 's feedback since we both paired on the implementation with @cdolm92 so I'm biased 😄

File: Kickstarter-iOS/Views/Controllers/SettingsViewController.swift [hunk 2]
```diff
@@ -89,6 +84,21 @@ final class SettingsViewController: UIViewController {
       .observeValues { [weak self] link in self?.goToAppStore(link: link) }
   }
 
+  private func leftBarButtonItem() -> UIBarButtonItem {
```
[the remark this answers is not in the document]
Comment (cdolm92) [reply Kickstarter-iOS/Views/Controllers/SettingsViewController.swift]: I didn't touch this file. I actually need to cherry pick to clean up this PR.

**Pull request closed without merging**